### PR TITLE
Solution: Manufacturing

### LINKED ISSUE #412: Add Manufacturing Solution page to website
### Story

_No response_

### Description

Navigation:
Add Solution to top navigation bar
Add Manufacturing to the submenu

Other solutions will be added over time

Title: Node-RED for Manufacturing
URL: https://flowforge.com/solution/manufacturing
Header Graphic: ideally a picture of a manufacturing equipment line

Text: https://docs.google.com/document/d/1EBpmv4sD68xMcDd0_tryfbGJ8yrMV2Ln7psd5fMmHww/edit?usp=sharing

Competitor pages:
https://www.crosser.io/solutions/industry-40/industrial-iot-data-integration/
https://www.crosser.io/solutions/industry-40/iot-edge-stream-analytics/
https://www.losant.com/solutions/industrial-equipment-monitoring

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- Add in a new navigation option: "Solutions"
- Adds the new `/solutions/manufacturing` page as per design & discussion in #412 

## Related Issue(s)

- Relies on #479 being merged
- Closes #412 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by joepavitt** [comment]
**Full Page & CTA Action:**

https://user-images.githubusercontent.com/99246719/222722564-055af7af-61c0-4133-853c-b88c60f57fa1.mov

**Mobile Sizing:**

https://user-images.githubusercontent.com/99246719/222722675-fdb3d74c-a041-45cc-8d79-e7bcdc3cb856.mov

**Comment by joepavitt** [comment]
Adding 2 Reviewers:
- @Pezmc to check this works locally, no errors, etc
- @iskerrett from a content/layout/design perspective

**Comment by iskerrett** [comment]
I can't get this branch to load locally. I get the following error messages.

[11ty] Image pipeline is enabled in dev mode expect a wait for first build
CssSyntaxError: postcss-import: C:\Users\iansk\Documents\GitHub\website\src\css\style.page.css:7:9: Unknown word

  5 | .page h1 {
  6 |     font-size: 2.5rem;
> 7 | <<<<<<< HEAD
    |         ^
  8 |     line-height: 4rem;
  9 | =======

[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] 1. Having trouble reading front matter from template ./src/solutions/manufacturing.njk (via TemplateContentFrontMatterError)
[11ty] 2. can not read a block mapping entry; a multiline key may not be an implicit key at line 6, column 8:
[11ty]     heroImg: ./images/pictograms/edge.png
[11ty]            ^ (via YAMLException)
[11ty]
[11ty] Original error stack trace: YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key at line 6, column 8:
[11ty]     heroImg: ./images/pictograms/edge.png
[11ty]            ^
[11ty]     at generateError (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:167:10)
[11ty]     at throwError (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:173:9)
[11ty]     at readBlockMapping (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1073:9)
[11ty]     at composeNode (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1359:12)
[11ty]     at readDocument (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1525:3)
[11ty]     at loadDocuments (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1588:5)
[11ty]     at load (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1614:19)
[11ty]     at Object.safeLoad (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1637:10)
[11ty]     at module.exports (C:\Users\iansk\Documents\GitHub\website\node_modules\gray-matter\lib\parse.js:12:17)
[11ty]     at parseMatter (C:\Users\iansk\Documents\GitHub\website\node_modules\gray-matter\index.js:109:17)
[11ty] Copied 17 files / Wrote 0 files in 0.44 seconds (v2.0.0)
ERROR: "dev:postcss" exited with 1.
PS C:\Users\iansk\documents\github\website>[11ty] Image pipeline is enabled in dev mode expect a wait for first build
CssSyntaxError: postcss-import: C:\Users\iansk\Documents\GitHub\website\src\css\style.page.css:7:9: Unknown word

  5 | .page h1 {
  6 |     font-size: 2.5rem;
> 7 | <<<<<<< HEAD
    |         ^
  8 |     line-height: 4rem;
  9 | =======

[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] 1. Having trouble reading front matter from template ./src/solutions/manufacturing.njk (via TemplateContentFrontMatterError)
[11ty] 2. can not read a block mapping entry; a multiline key may not be an implicit key at line 6, column 8:
[11ty]     heroImg: ./images/pictograms/edge.png
[11ty]            ^ (via YAMLException)
[11ty]
[11ty] Original error stack trace: YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key at line 6, column 8:
[11ty]     heroImg: ./images/pictograms/edge.png
[11ty]            ^
[11ty]     at generateError (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:167:10)
[11ty]     at throwError (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:173:9)
[11ty]     at readBlockMapping (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1073:9)
[11ty]     at composeNode (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1359:12)
[11ty]     at readDocument (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1525:3)
[11ty]     at loadDocuments (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1588:5)
[11ty]     at load (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1614:19)
[11ty]     at Object.safeLoad (C:\Users\iansk\Documents\GitHub\website\node_modules\js-yaml\lib\js-yaml\loader.js:1637:10)
[11ty]     at module.exports (C:\Users\iansk\Documents\GitHub\website\node_modules\gray-matter\lib\parse.js:12:17)
[11ty]     at parseMatter (C:\Users\iansk\Documents\GitHub\website\node_modules\gray-matter\index.js:109:17)
[11ty] Copied 17 files / Wrote 0 files in 0.44 seconds (v2.0.0)
ERROR: "dev:postcss" exited with 1.
PS C:\Users\iansk\documents\github\website>

**Review by iskerrett — APPROVED** [review]

File: src/_includes/layouts/base.njk [hunk 1]
```diff
@@ -101,6 +101,12 @@
                     <ul id="nav-content" class="hidden absolute top-8 right-0 border shadow-sm md:inline md:shadow-none w-36 z-10 md:border-0 bg-white md:bg-transparent md:w-auto rounded md:static md:float-none md:flex md:flex-row flex-col items-center justify-between font-medium text no-underline">
                         <li class="flex"><a href="/features">features</a></li>
                         <li class="flex"><a href="/pricing">pricing</a></li>
+                        <li class="ff-nav-dropdown relative">
+                            <a class="flex items-center gap-2">solutions {% include "components/icons/chevron-down.svg" %}</a>
```
Comment (Pezmc) [inline src/_includes/layouts/base.njk:105]: FYI this _could_ use the `{% image` helper, but we already have this convention in this file so no need to change it now

File: src/solutions/manufacturing.njk [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+                        that can be easily modified and managed.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="w-full md:max-w-screen-lg m-auto pb-32">
+    {% image "./images/solutions/markitechture-manufacturing.jpg", "", [1024] %}
```
Comment (Pezmc) [inline src/solutions/manufacturing.njk:86]: Add an alt tag for SEO + accessibility

**Comment by Pezmc** [comment]
(Merged before approving, 👍)

**Review by Pezmc — COMMENTED** [review]
Checked out locally, looks good and builds without issues 👍

**Pull request merged**